tests/e2e/util.go

```diff
@@ -1,10 +1,34 @@
 package e2e
 
 import (
+	"context"
+	"fmt"
 	"os"
+	"os/exec"
+	"strings"
 
 	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	. "github.com/onsi/ginkgo/v2" //nolint:stylecheck,revive
+	v1 "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/meta/v1"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	v1extensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	errKymaDeployCommandFailed  = errors.New("failed to run kyma alpha deploy")
+	errModuleTemplateNotApplied = errors.New("failed to apply ModuleTemplate")
+	errModuleEnablingFailed     = errors.New("failed to enable module")
+	errModuleDisablingFailed    = errors.New("failed to disable module")
+)
+
+const (
+	exitCodeNoError = 0
+	exitCodeWarning = 2
+	readyState      = v1beta2.StateReady
```

Comment:
do we really need to define this const value specifically? why not directly use v1beta2.StateReady in the code? 

Basically const value have two purpose:
1. give a meaningful name to a value, here the meaning of `readyState` and `StateReady` is the same to me, I don't need this const value for me to understand better.
2. reusable, and since we already defined State in v1beta2 package, we can use it directly anywhere.

Reply:
ah make sense